doc/ros_visualization/visualization_tutorial.rst

```diff
@@ -34,6 +28,13 @@ Build and source the moveit_config package::
 
 *(NOTE: If you don't use the catkin package, simply replace 'catkin build' with 'catkin_make')*
 
+Alternatively, you can complete the `MoveIt! Setup Assistant tutorial
+<../setup_assistant/setup_assistant_tutorial.html>`_
+to generate a MoveIt! configuration for the PR2 that you
+can use.  This tutorial assumes the generated MoveIt! configuration
+package is called "pr2_moveit_config".
+
+
```

Comment:
shoot a new user had this same issue today so i clarified it for them here:

https://github.com/ros-planning/moveit_tutorials/pull/126/files

I think mine is a bit more in-depth in explaining the options - what do you think? perhaps remove your changes at the top of the file but keep the rest in this PR?

Reply:
I agree, your wording is better. I'll revert that section and push the changes.